modules/developers-guide/examples/javascript/node-gen-table-token.js

```diff
@@ -0,0 +1,17 @@
+// Stargate OSS configuration for locally hosted docker image
+const auth_endpoint = "http://localhost:8081/v1/auth";
+const username = "cassandra";
+const password = "cassandra";
+const stargate_uri = "localhost:8090";
+
+// Set up the authenication
+// For Stargate OSS: Create a table based auth token Stargate/Cassandra
+// authentication using the default C* username and password
+const credentials = new StargateTableBasedToken(
+  {authEndpoint: auth_endpoint,
+    username: username,
+    password: password
+  }
+);
+
+console.log(credentials);
```

Comment:
Do we want to keep the log statements for objects like these credentials? To me it makes sense to log strings like "table created," "keyspace created" etc. but this is going to dump a big ol' object to the screen that may not provide the user much value. But I don't have much experience writing docs, so I'd defer to you on this - what do you think?

Reply:
Left in, but added comment about uncommented for use.